server/src/database/board_overview.go

```diff
@@ -0,0 +1,28 @@
+package database
+
+import "github.com/google/uuid"
+
+func (d *Database) GetBoardOverview(id uuid.UUID) (Board, []BoardSession, []Column, error) {
```

Comment:
Why don't you return `BoardOverview` from the `common/dto` package here instead of every ingredient individually?

Reply:
That could theoretically be done, but we would break with the model that was implemented otherwise. We usually receive all the database objects, transform them into a data transfer objects inside the service package and then provide them through the api package. 
I see though, that the BoardOverview should probably be created inside the service package and not inside the api